test/instrumentation/azure-functions/azure-functions-v3.test.js

```diff
@@ -168,7 +52,7 @@ function checkExpectedApmEvents(t, apmEvents) {
   if (apmEvents.length > 0) {
     const metadata = apmEvents.shift().metadata;
     t.ok(metadata, 'metadata is first event');
-    t.equal(metadata.service.name, 'AJsAzureFnApp', 'metadata.service.name');
+    t.equal(metadata.service.name, 'azfunc3', 'metadata.service.name');
```

Comment:
Reviewer note: This and all the remaining diffs in this file are from renaming the fixture: s/AJsAzureFnApp/azfunc3/.